### PR TITLE
config: Extend documentation of CDASH_DAILY_UPDATES

### PR DESCRIPTION
**Pull request by jcfr**
This commit follows up ae2de8a (Allow daily updates to be triggered outside of submissions)

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -53,8 +53,16 @@
 $CDASH_TESTING_RENAME_LOGS = false;
 
 // Whether or not CDash submissions should trigger daily updates.
+//
 // Disable this if you want more fine grained control over when/how
 // daily updates are triggered (e.g. cron).
+//
+// Note: Disabling the daily updates is useful to avoid costly call to
+// `addDailyChanges` implemented in `include/dailyupdates.php`. In
+// situation where a lot of submission are processed (e.g > 10000/days),
```
Comment (jcfr) [inline config/config.php]: @danlamanna I chose `> 10000/days` arbitrarily

Comment (jamiesnape) [inline config/config.php]: I would omit a number. Technically, this is an improvement for even a small number of submissions.

Comment (zackgalbreath) [inline config/config.php]: > this is an improvement for even a small number of submissions

Assuming you're willing to maintain a cronjob to do this instead (or live without it).

Comment (jamiesnape) [inline config/config.php]: Or set an alarm to remind you to curl it yourself?

**Pull request closed without merging**